Services/Services_bases/telegram_api_service/telegram_api.py

```diff
@@ -0,0 +1,147 @@
+               and services_constants.CONFIG_SERVICE_INSTANCE in config[services_constants.CONFIG_CATEGORY_SERVICES][
+                   services_constants.CONFIG_TELEGRAM_API]
+
+    async def prepare(self):
+        if not self.telegram_client:
+            try:
+                self.telegram_client = telethon.TelegramClient(f"{common_constants.USER_FOLDER}/telegram-api",
+                                                               self.config[services_constants.CONFIG_CATEGORY_SERVICES]
+                                                               [services_constants.CONFIG_TELEGRAM_API]
+                                                               [services_constants.CONFIG_API],
+                                                               self.config[services_constants.CONFIG_CATEGORY_SERVICES]
+                                                               [services_constants.CONFIG_TELEGRAM_API]
+                                                               [services_constants.CONFIG_API_HASH],
+                                                               base_logger=self.get_name())
+
+                await self.telegram_client.start(
+                    phone=
+                    self.config[services_constants.CONFIG_CATEGORY_SERVICES][services_constants.CONFIG_TELEGRAM_API]
+                    [services_constants.CONFIG_TELEGRAM_PHONE]
+                )
+                self.user_account = await self.telegram_client.get_me()
+                self.connected = True
+            except Exception as e:
+                self.logger.error(f"Failed to connect to Telegram Api : {e}")
+
+    def is_running(self):
+        return self.telegram_client.is_connected()
+
+    def get_type(self):
+        return services_constants.CONFIG_TELEGRAM_API
+
+    def get_website_url(self):
+        return "https://telegram.org/"
+
+    def get_endpoint(self):
+        return self.telegram_client
+
+    def get_brand_name(self):
+        return "telegram"
+
+    def stop(self):
+        if self.connected:
+            self.telegram_client.disconnect()
+            self.connected = False
+
+    @staticmethod
+    def get_is_enabled(config):
+        return services_constants.CONFIG_CATEGORY_SERVICES in config \
+               and services_constants.CONFIG_TELEGRAM_API in config[services_constants.CONFIG_CATEGORY_SERVICES]
+
+    def has_required_configuration(self):
+        return services_constants.CONFIG_CATEGORY_SERVICES in self.config \
+               and services_constants.CONFIG_TELEGRAM_API in self.config[services_constants.CONFIG_CATEGORY_SERVICES] \
+               and self.check_required_config(
+            self.config[services_constants.CONFIG_CATEGORY_SERVICES][services_constants.CONFIG_TELEGRAM_API]) \
+               and self.get_is_enabled(self.config)
+
+    async def send_message(self, content, markdown=False, reply_to_message_id=None) -> telegram.Message:
```

Comment:
when is this used ?

Reply:
It's not used for now, do you think we should remove it ?